src/js/dropdown-menu/specs/dropdown-menu-view-spec.js

```diff
@@ -1,9 +1,11 @@
 define([
         'jquery',
         'underscore',
-        '../dropdown-menu-view.js'
+        '../dropdown-menu-view.js',
+        '../../utils/constants.js',
+        '../../../../test/jquery.simulate.js'
```

Comment:
Nit: I'd refer to this via an absolute path. IMO relative paths only make sense if there is a relationship between the two files. This relative path would break if we happened to move the dropdown menu code for some reason.